shared/components/src/Index.tsx

```diff
@@ -0,0 +1,3 @@
+import * as React from 'react';
+
+export { VisualIdentifier } from './visual-identifier/VisualIdentifier';
```

Comment:
Just a question. For each shared component, will we need to add an export here?

Reply:
Yes! 
Shared Components should remain in special cases, we shouldn't have too many of them.